src/components/home/ventures_section/portfolio_subsection.js

```diff
@@ -8,9 +8,10 @@ import Venture from "./portfolio_subsection/venture"
 import styles from "./portfolio_subsection.module.css"
 
 const PortfolioSubsection = ({ planetMorph, ventures }) => {
+  const baseDelay = 0.3
```

Comment:
Why this change? `baseDelay` is not being used outside `renderVenture` is it?

Reply:
I was going to move it outside the component. Just because it's a constant.